packages/core/src/external/carequality/ihe-gateway-v2/inbound/xcpd/process/xcpd-request.ts

```diff
@@ -84,7 +84,7 @@ export async function processInboundXcpdRequest(
     return inboundRequest;
   } catch (error) {
     const msg = "Failed to parse ITI-55 request";
-    log(`${msg}: Error - ${errorToString(error)}`);
+    log(`${msg}: Error - ${errorToString(error)}, JsonObj: ${JSON.stringify(jsonObj)}`);
```

Comment:
better logging for when we error

Reply:
nit: `JsonObj` is not very descriptive here. What does it represent?